projects/packages/identity-crisis/src/class-identity-crisis.php

```diff
@@ -0,0 +1,1026 @@
+		 * JETPACK_SYNC_IDC_OPTIN constant if set.
+		 *
+		 * @param bool $default Whether the site is opted in to IDC mitigation.
+		 *
+		 * @since 4.3.2
+		 */
+		return (bool) apply_filters( 'jetpack_sync_idc_optin', $default );
+	}
+
+	/**
+	 * Does the current admin page have help tabs?
+	 *
+	 * @return bool
+	 */
+	public function admin_page_has_help_tabs() {
+		if ( ! function_exists( 'get_current_screen' ) ) {
+			return false;
+		}
+
+		$current_screen = get_current_screen();
+		$tabs           = $current_screen->get_help_tabs();
+
+		return ! empty( $tabs );
+	}
+
+	/**
+	 * Renders the non-admin IDC notice.
+	 *
+	 * @return void
+	 */
+	public function display_non_admin_idc_notice() {
+		$classes = 'jp-idc-notice inline is-non-admin notice notice-warning';
+		if ( isset( self::$current_screen ) && 'toplevel_page_jetpack' !== self::$current_screen->id ) {
+			$classes .= ' is-dismissible';
+		}
+
+		if ( $this->admin_page_has_help_tabs() ) {
+			$classes .= ' has-help-tabs';
+		}
+		?>
+
+		<div class="<?php echo esc_attr( $classes ); ?>">
+			<?php $this->render_notice_header(); ?>
+			<div class="jp-idc-notice__content-header">
+				<h3 class="jp-idc-notice__content-header__lead">
+					<?php echo esc_html( $this->get_non_admin_notice_text() ); ?>
+				</h3>
+
+				<p class="jp-idc-notice__content-header__explanation">
+					<?php echo esc_html( $this->get_non_admin_contact_admin_text() ); ?>
+				</p>
+			</div>
+		</div>
+		<?php
+	}
+
+	/**
+	 * First "step" of the IDC mitigation. Will provide some messaging and two options/buttons.
+	 * "Confirm Staging" - Dismiss the notice and continue on with our lives in staging mode.
+	 * "Fix Jetpack Connection" - Will disconnect the site and start the mitigation...
+	 *
+	 * @return void
+	 */
+	public function display_idc_notice() {
+		$classes = 'jp-idc-notice inline notice notice-warning';
+		if ( $this->admin_page_has_help_tabs() ) {
+			$classes .= ' has-help-tabs';
+		}
+		?>
+		<div class="<?php echo esc_attr( $classes ); ?>">
+			<?php $this->render_notice_header(); ?>
+			<?php $this->render_notice_first_step(); ?>
+			<?php $this->render_notice_second_step(); ?>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Enqueue CSS for the admin bar.
+	 *
+	 * @return void
+	 */
+	public function enqueue_admin_bar_css() {
+		wp_enqueue_style(
+			'jetpack-idc-admin-bar-css',
+			plugins_url( '_inc/build/jetpack-idc-admin-bar.css', JETPACK__PLUGIN_FILE ),
+			array( 'dashicons' ),
+			JETPACK__VERSION
+		);
+	}
+
+	/**
+	 * Enqueue scripts for the notice.
+	 *
+	 * @return void
+	 */
+	public function enqueue_idc_notice_files() {
+		$build_assets                   = require_once __DIR__ . '/../build/index.asset.php';
+		$build_assets['dependencies'][] = 'jquery';
+
+		wp_enqueue_script(
+			'jetpack-idc-js',
+			Assets::get_file_url_for_environment( '_inc/build/idc-notice.min.js', '_inc/build/idc-notice.js' ),
```

Comment:
this call is wrong.

Path is _inc/idc-notice.js for both min and non-min it also needs to pass the path.